components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.html

```diff
@@ -3,15 +3,20 @@
   </div>
   <ul class="bx--row">
     <ng-container *ngFor="let option of options">
-      <div class="filter-button-container">
-        <li class="filter-button" (click)="handleSelect($event)" (keyup.enter)="handleSelect($event)"
-        tabindex="0" role="button" [attr.aria-pressed]="false" [attr.data-filtervalue]="option.id">
+      <li class="filter-button-container">
+        <div class="filter-button"
+          (click)="handleSelect($event)"
+          (keyup.enter)="handleSelect($event)"
+          tabindex="0"
+          role="button"
+          [attr.aria-pressed]="false"
+          [attr.data-filtervalue]="option.id">
```

Comment:
single line per attribute for easier reading.

while in here I also noticed I had `div` elements directly nested in `ul`s, so I swapped them for better semantic html